src/Styleguide/Elements/Card.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components"
+import { Flex } from "./Flex"
+import { themeGet } from "styled-system"
+
+export const Card = styled(Flex)`
```

Comment:
I looked at the `themeGet` implementation and he's not using lodash, so that won't work unfortunately, but _can_ do `themeGet('space')[4]`. I'll go ahead and merge for now and we can loop back. (Also fine to just do `padding: 20px` with a lil note.) By the time we're done with this i think we'll have the space values pretty well internalized :)